src/main/java/ch/njol/skript/util/VisualEffect.java

```diff
@@ -159,17 +159,16 @@ public boolean isColorable() {
 		ITEM_CRACK(Particle.ITEM_CRACK) {
 			@Override
 			public Object getData(final @Nullable Object raw, final Location l) {
-				if (raw == null)
-					return Material.IRON_SWORD;
-				else if (raw instanceof ItemType) {
+				ItemStack itemStack = new ItemStack( Material.IRON_SWORD);

```

Comment:
Even if this Material didn't change between versions, it might be better to have a constant with `Aliases.javaItemtype("iron sword")` and use ItemType#getMaterial here, just to avoid referencing the enum in general.

Reply:
Good idea... never know what is going to happen in the future.

Reply:
Updated :)